artifacts_test.py

```diff
@@ -262,6 +272,10 @@ def test_scylla_service(self):
         with self.subTest("verify write cache for NVMe devices"):
             self.verify_nvme_write_cache()
 
+        if not self.params["use_preinstalled_scylla"]:
```

Comment:
let's enable the check for all except Docker

Reply:
And this is exactly what happened.